src/TLuaInterpreter.cpp

```diff
@@ -949,6 +949,22 @@ int TLuaInterpreter::getTextFormat(lua_State* L)
     return 1;
 }
 
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#getWindowsCodepage
+int TLuaInterpreter::getWindowsCodepage(lua_State* L)
+{
+#if defined (Q_OS_WIN32)
+    QSettings registry(QStringLiteral(R"(HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Nls\CodePage)"),
+                       QSettings::NativeFormat);
+    auto value = registry.value(QStringLiteral("ACP"));
+    lua_pushstring(L, value.toString().toUtf8().constData());
+    return 1;
+#else
```

Comment:
I would comment that this will need checking for Cygwin builds but since that is not an environment we currently pursue I guess the person who tries a port can find that out for themselves. :smiling_imp: